### PR TITLE
Fix crontab newline issue on wcoss_dell_p3

### PR DESCRIPTION
**Pull request by chan-hoo**
## DESCRIPTION OF CHANGES: 
- Fix the issue that an additional crontab command is not written in a new line on WCOSS Dell.
- Clean up WCOSS Cray and unnecessary modules for WCOSS Dell.
- Remove the `docs` directory from the regional workflow.

## TESTS CONDUCTED: 
WE2E tests on WCOSS_DELL_P3 and Hera:
- grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
- grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thompson_mynn_lam3km
- MET_verification

### CODE REVIEW

File: docs/UsersGuide/source/Chapter2.rst [hunk 1]
```diff
@@ -214,7 +214,7 @@ Note the following:
 
 * The second argument to the ``compile.sh`` script is ``theia.intel``, not just ``theia``. 
     Other build targets are: 
-    ``cheyenne.gnu, cheyenne.intel, cheyenne.pgi, odin, theia.gnu, theia.intel, theia.pgi`` or ``wcoss_cray``.
+    ``cheyenne.gnu, cheyenne.intel, cheyenne.pgi, odin, theia.gnu, theia.intel`` or ``theia.pgi``.
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/Chapter2.rst:218]: Do we really have PGI support on Theia?

Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/Chapter2.rst]: I thought we got rid of the docs directory in regional_workflow in favor of housing all documentation in ufs-srweather-app?  Also, Theia's been retired for a long time.

Comment (chan-hoo) [reply docs/UsersGuide/source/Chapter2.rst]: @JeffBeck-NOAA, yes.. I remember that we removed this docs directory in my previous PR which was closed without merged. I'll remove the entire docs directory in this PR again.

Comment (chan-hoo) [reply docs/UsersGuide/source/Chapter2.rst]: Done.

**Review by gsketefian — APPROVED** [review]

**Pull request merged**